src/streamlink/plugins/liveme.py

```diff
@@ -26,6 +27,16 @@ class LiveMe(Plugin):
     def can_handle_url(cls, url):
         return cls.url_re.match(url) is not None
 
+    def _random_t(self, t):
+        n = "ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678"
+        e = len(n)
+        a = ""
+        r = 0
+        while (r < t):
+            a += n[math.floor(random.randrange(e))]
```

Comment:
`random.choice` might be a better method to use, eg. `random.choice(n)`. 
Personally, I'd just use a list comprehension/generator to do it :) 
```python
def _random_t(self, t):
    return "".join(random.choice("ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678") for _ in range(t))
```

Reply:
i just did the same, as it is on their website.

```js
function t(t) {
    t = t || 32;
    for (var n = "ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678", e = n.length, A = "", r = 0; r < t; r++) A += n.charAt(Math.floor(Math.random() * e));
    return A
}
```

but `random.choice` looks easier.